proto/wg/cosmo/platform/v1/platform.proto

```diff
@@ -2779,6 +2798,10 @@ service PlatformService {
   rpc GetCacheWarmerOperations(GetCacheWarmerOperationsRequest) returns (GetCacheWarmerOperationsResponse) {}
   // ComputeCacheWarmerOperations computes the list of cache warmer operations.
   rpc ComputeCacheWarmerOperations(ComputeCacheWarmerOperationsRequest) returns (ComputeCacheWarmerOperationsResponse) {}
+  // EnableCacheWarmer enables cache warmer for the namespace passed.
+  rpc EnableCacheWarmer(EnableCacheWarmerRequest) returns (EnableCacheWarmerResponse) {}
```

Comment:
Do you also disable via this RPC? If yes, I'd rename this to "ConfigureCacheWarmer"

Reply:
Done